samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,20 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: ci_build ci_rs_weekly
+  sample.peripheral.802154_phy_test.ant_div_mode_auto:
+    build_only: true
+    integration_platforms:
+      - nrf21540dk_nrf52840
+    platform_allow: nrf21540dk_nrf52840
+    tags: ci_build ci_rs_weekly
+    extra_args: DCONFIG_PTT_ANTENNA_DIVERSITY=y
```

Comment:
Remove `D` prefix from any Kconfig here, twister will add it underneath. please also manually verify via twister if all samples could be builded using zephyr twister tool

Reply:
I removed the prefixes, manual build works :)